src/pages/home/subscription/subscription.styled.js

```diff
@@ -1,10 +1,78 @@
 import styled from 'styled-components';
-import { colors } from '@tidb-community/ui';
+import { colors, mixins } from '@tidb-community/ui';
 
 import { Section } from '~/pages/home/index.styled';
+import { Button, Input, Row } from 'antd';
 
-export const Container = styled(Section)`
-  && {
-    background: ${colors.M2};
+export const Container = styled.div`
+  ${mixins.reset()}
+  ${mixins.centerBlock()}
+  ${mixins.responsive()}
+
+  ${(props) =>
+    props.isMobile &&
+    `
+    text-align: center;
+    ${mixins.flexCenter()}
```

Comment:
Not a big deal, but normally I perfer adding a colon at the end of each statement.